static/js/lib/api/api.js

```diff
@@ -34,6 +35,27 @@ export function search(params: SearchParams): Promise<*> {
   })
 }
 
+export function aggregate(fields: Array<string>): Promise<*> {
+  const aggs = _.fromPairs(
+    fields.map(field => [
+      field,
+      {
+        terms: {
+          field: field,
+          size:  100000
```

Comment:
What does this value mean here?

Reply:
It's a limit on how many results to return, I set it very high to make sure all topics were included.